src/System.Memory/System.Memory.sln

```diff
@@ -25,24 +25,16 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{E107E9C1-E89
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ref", "ref", "{2E666815-2EDB-464B-9DF6-380BF4789AD4}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{3F794AE4-319F-4498-A9DA-C49A4EB11D9E}"
-	ProjectSection(SolutionItems) = preProject
-		..\Common\src\CoreLib\System\Runtime\InteropServices\MemoryMarshal.Fast.cs = ..\Common\src\CoreLib\System\Runtime\InteropServices\MemoryMarshal.Fast.cs
```

Comment:
Can we keep these files in the solution? They are coming from corelib but are complimentary to the "portable" versions of the files and it makes things a lot easier to have them visible from the solution.

Reply:
There isn't any reason to not have them but they don't easily regenerate when we refresh the solution files. We have the generator to keep the solution consistent and automatically create-able. You can add them back but you will be fighting the auto-generation unless we can come up with a way to have that round trip correctly.

Reply:
How often do we auto-generate the sln files? I get that it could be cumbersome to create an exception for changes like this. If the updates are relatively infrequent, I might consider adding them back.

Reply:
Instead of the sln file, could you add them as a ‘None’ item to a .csproj?

Reply:
From https://msdn.microsoft.com/library/0c6xyb66(v=vs.100).aspx:
> None - The file is not included in the project output group and is not compiled in the build process. An example is a text file that contains documentation, such as a Readme file.

Yes, that would work!